modules/monitoring-basic-platform-monitoring-configuration-scenario.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-platform-monitoring-basic-configuration-scenario_{context}"]
+= Platform monitoring: Basic configuration scenario
+
+This scenario describes setup steps to help administrators get started with monitoring services in a {product-title} cluster using the in-cluster monitoring stack.
+These setup steps include configuring ServiceMonitors, metrics, alerts, and persistent storage.
+
+.Prerequisites
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created users.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Set permissions for users:
+
+.. Allow users to create alerts:
++
+[source,terminal]
+----
+$ oc policy add-role-to-user monitoring-edit <user> -n <namespace>
+----
++
+.. Allow users to route alerts:
++
+[source,terminal]
+----
+$ oc -n <namespace> adm policy add-role-to-user alert-routing-edit <user>
+----
++
+. If it does not exist, create the `cluster-monitoring-config` `ConfigMap` object.
```

Comment:
Would it be worth creating a note about the plan to move this to CRDs in future releaseS?

Reply:
> Would it be worth creating a note about the plan to move this to CRDs in future releaseS?

@danielmellado - As a rule, we don't talk about future plans for features because both the features and the timelines are subject to change. The earliest we'd mention something is if it is implemented in the product as a developer or tech preview feature.